data/strings/types_strings.txt

```diff
@@ -26875,6 +26875,51 @@
     zh-Hans = 自提点
     zh-Hant = 接送的地點
 
+  [type.shop.pasta]
+    en = Pasta
```

Comment:
Actually it could be easy to confuse it with an eatery that specializes in serving pasta.

Should it be a more explicit "Pasta Shop"?